VanDerPol/FMI2.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <fmiModelDescription
   fmiVersion="2.0"
-  modelName="Van der Pol oscillator"
-  description="This model implements the van der Pol oscillator"
+  modelName="van der Pol oscillator"
```

Comment:
Model names should stay capitalized for consistency.

Reply:
OK, then it should be changed in all other places instead of this one.